bundles/org.smarthomej.binding.deconz/src/main/resources/OH-INF/config/config.xml

```diff
@@ -32,6 +32,12 @@
 			<advanced>true</advanced>
 			<default>2000</default>
 		</parameter>
+		<parameter name="websocketTimeout" type="integer" unit="s" min="30">
+			<label>Websocket Idle Timeout</label>
```

Comment:
Should we label the two timeouts in "HTTP Timeout" and "Websocket Timeout"? Similar to the port.

Alternatively we can introduce parameter-groups.